lib/jekyll-github-metadata/repository_finder.rb

```diff
@@ -46,15 +48,19 @@ def nwo_from_config
       end
 
       def git_exe_path
+        exts = (ENV["PATHEXT"] || "").split(File::PATH_SEPARATOR).push("")
+        cmds = exts.map { |ext| "git#{ext}" }
```

Comment:
Is there any known scenario where `git.exe` is **not the Git Executable** on Windows? If not, lets not waste time searching for those paths and zero in on `~/git.exe` straight away.

Reply:
I don't really have visibility into how people have their machines setup, but having a `git.bat` file to wrap Git is one edge case I can think of.

Random example: https://gist.github.com/duebbert/c4f26c9d0691b7d4a0e932e7d9f36271

Technically all of the extensions in `PATH_EXT` can be used plainly as `git` from the command line on Windows if someone added a wrapper or alternate implementation to their `PATH`.

If you want to simplify it, I recommend skipping the `PATH` search altogether and allowing the `Open3`/`Exec.run` call to outsource that to the OS.

Do you know why this `PATH` code exists in the first place?

Reply:
Delegating to Windows can be bothersome IMO.
Some machines respond to `where` while some responds to `where.exe` while the others do not have either of the two programs in their `PATH`..